lizard_auth_client/resources.py

```diff
@@ -0,0 +1,17 @@
+from import_export import resources
+from lizard_auth_client import models
+
+
+class RoleResource(resources.ModelResource):
+    class Meta:
+        model = Role
```

Comment:
You'll need to add some tests :-)

This line cannot work: it should be `models.Role`. So run "pyflakes" or "flake8", that'll report stuff like this.

And, yes, could you add a simple test that checks if the new functionality works a bit? The test coverage went down and that's not good for an essential package used by all our sites.

(Note to self: I should take a look why the coverage is so low. I think lizard-auth-server's is much higher.)

Reply:
Perhaps you can add a pyflakes/flake8 step to the travis/jenkins test run?

Reply:
I added it but still I need to correct some flake8 errors...